runtime/onert/core/src/exec/MinMaxData.h

```diff
@@ -30,6 +30,8 @@ namespace exec
 // we need to dump model, subgraph id for each minmax
 
 // File structure
+// uint32_t magic code
+// uint32_t version
```

Comment:
(optional) I suggested the acceptable minimal size of types. Though it is just a few bytes, it may be a precious ones on a few kilobytes devices. I assumes the minmax format can be same for onert-micro.

In addition, not only magic code and version, but also for other fields in minmaxdump.

Again, it is just my optional suggestion.